### DISCUSSION TITLE
X6的2.0版本进度如何了？

### DISCUSSION THREAD

**Discussion by xiazhaohui** [antvis/X6#2896] [General]
hello 我了解到目前在做升级迭代的事情，性能会有较大的提升，X6官方在语雀的文档有做对应的记录
https://www.yuque.com/antv/x6/sk3zdn_nsgg4p

<img width="759" alt="image" src="https://user-images.githubusercontent.com/82207908/202374990-b6b2e2fd-3e4a-421b-b6ef-3541b7dadf75.png">

进度大概是如下图所示的样子：
<img width="1084" alt="image" src="https://user-images.githubusercontent.com/82207908/202375110-5a378703-8865-4335-9de7-9953a98149b2.png">

我想咨询一下，目前有没有一个具体的迭代计划，2.0版本大概能什么时候上线呢？xdm应该都还挺期待
⚽️

**Comment by xiazhaohui** [comment]
看来是这个月22号
https://www.yuque.com/antv/blog/kt8nugz3kdg32h7v
![image](https://user-images.githubusercontent.com/82207908/202982774-74dc800c-b443-4e79-847e-95079b8c333c.png)

**Reply by NewByVector** [reply]
是的 今晚发布新版本。

**Reply by xiazhaohui** [reply]
> 是的 今晚发布新版本。

四大升级？看来除了提升性能外还有新特性啊，兼容之前版本所做的功能吗

**Reply by xiazhaohui** [reply]
> 是的 今晚发布新版本。

几点？有发布会或者视频会议吗，看一下

**Reply by NewByVector** [reply]
兼容之前功能的。不过没有视频会议。

**Reply by xiazhaohui** [reply]
> 兼容之前功能的。不过没有视频会议。

这样啊，是直接更新的官网和API文档吗

**Reply by NewByVector** [reply]
官网会更新一波，不过用法改动不大。
